libaugrim/src/time.rs

```diff
@@ -13,10 +13,11 @@
 // limitations under the License.
 
 use std::cmp::PartialOrd;
+use std::fmt::Debug;
```

Comment:
This is part of the standard prelude, so you shouldn’t have to explicitly use it

Reply:
That does not seem to be the case when used like this, as I get a compiler "not a trait" error without this line.